monitor/dbt_project/models/alerts/alerts_models.sql

```diff
@@ -2,17 +2,16 @@
   config(
     materialized = 'incremental',
     unique_key = 'alert_id',
-    merge_update_columns = ['alert_id']
+    merge_update_columns = ['alert_id'],
+    on_schema_change = 'append_new_columns'
   )
 }}
 
 
-with alerts_dbt_models as (
-    select * from {{ ref('elementary', 'alerts_dbt_models') }}
-)
-
-select *, false as alert_sent from alerts_dbt_models
-
-{%- if is_incremental() %}
-    {{ get_new_alerts_where_clause(this) }}
-{%- endif %}
+{% set error_models_relation = adapter.get_relation(this.database, this.schema, 'error_models') %}
```

Comment:
I'd check also if relation_exists and also if not, you need to return a select statement with the same schema so the macro that reads it will work (see empty_table.sql as a reference)

Reply:
It [seems](https://github.com/dbt-labs/dbt-core/blob/8442fb66a591c4f5353f8baaaaf62f846c5f5171/core/dbt/include/global_project/macros/adapters/relation.sql#L70) that `load_relation()` is just a user-friendly way of invoking `get_relation()`.  
Could you please elaborate a bit on why do it?